src/app/core/components/common/data-table/data-table.component.html

```diff
@@ -1,11 +1,19 @@
 @if (rows.length) {
   <div [style.max-height]="maxHeight" [class.scrollable]="maxHeight">
     <table class="govuk-table" data-module="moj-sortable-table">
-      <caption class="govuk-table__caption govuk-table__caption--m">
-        {{
-          caption
-        }}
-      </caption>
+      @if (hiddenCaption) {
+        <caption class="govuk-table__caption govuk-table__caption--m govuk-visually-hidden">
+          {{
+            caption
+          }}
+        </caption>
+      } @else {
+        <caption class="govuk-table__caption govuk-table__caption--m">
+          {{
+            caption
+          }}
+        </caption>
+      }
```

Comment:
Instead of using a full-blown if else statement for this, you could use:

`[class.govuk-visually-hidden]="hiddenCaption"`

Reply:
changes incoming!